apps/OboeTester/app/src/main/cpp/NativeAudioContext.cpp

```diff
@@ -65,17 +68,27 @@ void NativeAudioContext::connectTone() {
         switch (mToneType) {
             case ToneType::SawPing:
                 sawPingGenerator.output.connect(&(monoToMulti->input));
-                monoToMulti->output.connect(&(audioStreamGateway.get()->input));
+                monoToMulti->output.connect(&(mSinkFloat.get()->input));
```

Comment:
Feels like duplication throughout this switch block. Could you just use a float flowgraph and have a float->int16 conversion at the final output if the stream type is I16 (as per the recent update to the samples)?

Reply:
I could factor out the manyToMulti vs monoToMulti code.

Reply:
I looked at refactoring but it did not seem cleaner. There is no code common to every case.